packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-source.tsx

```diff
@@ -0,0 +1,104 @@
+import {Component, Prop, h, Listen} from '@stencil/core';
+import {buildInteractiveResult, Result, SmartSnippet} from '@coveo/headless';
+import {LinkWithResultAnalytics} from '../result-link/result-link';
+import {
+  Bindings,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import {ResultContextEvent} from '../result-template-components/result-template-decorators';
+
+@Component({
```

Comment:
We use a JSDoc tag (internal) so the documentation for this component is not generated in the reference doc https://docs.coveo.com/en/atomic/latest/reference/
```suggestion
/**
 * @internal
 */
@Component({
```